docusaurus/docs/cloud/getting-started/deployment.md

```diff
@@ -0,0 +1,76 @@
+
+### Create a project
+
+The first time logging into Strapi Cloud, a welcome page is displayed providing a link to the terms of service and prompting you to **Start free trial** to continue.
+
+![Strapi Cloud welcome page](/img/assets/cloud/welcome-screen.png)
+
+1. From the **Projects** page, click the **Create Project** button. You are prompted to **Connect with GitHub**.
+
+    :::tip
+    Connect the GitHub account and/or Organizations that own the repository or repositories you want to deploy. This can be different from the account that owns the Strapi Cloud account.
+
+    You will be redirected to GitHub to authorize Strapi Cloud to access your repository.
+    :::
+
+2. After granting the required access from GitHub, from the **Projects** page select your desired repository to install Strapi Cloud.
+
+    ![Project Import - Select Repository](/img/assets/cloud/import.png)
+
+3. Click **Next** to proceed to the Project Set up page and enter the following information:
+    * **Project name**: The name of your Strapi app, this is fetched from the repository name but can be edited. It is automatically converted to slug format (`my-strapi-app`).
+    * **GitHub branch**: The default branch to use for this deployment. This uses the [default branch](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/changing-the-default-branch) of the repository but can be changed via the drop-down.
+    * **Deploy on push**: When enabled, this will automatically deploy the latest changes from the selected branch. When disabled, you will need to manually deploy the latest changes.
+
+    ![Project Setup](/img/assets/cloud/setup.png)
+
+4. (**Optional**) Select **Show Advanced Settings** to configure the following options:
+    * **Base directory**: The directory where your Strapi app is located in the repository. This is useful if you have multiple Strapi apps in the same repository or if you have a monorepo.
+    * [**Environment variables**](/dev-docs/configurations/environment/): Environment variables are used to configure your Strapi app. You can add environment variables to your Strapi Cloud project by clicking **Add Environment Variable**. You can also add environment variables to your Strapi app by adding a `.env` file to the root of your Strapi app directory. The environment variables defined in the `.env` file will be used by Strapi Cloud.
+
+    ![Advanced Setup](/img/assets/cloud/advanced.png)
+
+5. Click **Next** to proceed to the **Payment and billing** page. Enter the corresponding billing details.
+
+    :::tip
+    This is for verification purposes only. You will not be charged during the Beta release, or without notice.
+    :::
+
+    ![Payment and billing](/img/assets/cloud/billing-info.png)
+
+6. Click **Create** to finalize the project creation. An initial deployment is triggered and you are redirected to the **Projects** page.
```

Comment:
@gpene could we please add here (or even better if we can add it as some warning/tip) something like this: `Please create your first admin user in Strapi once your deployment is finished & do not share your app's URL before that.`

**Some context:** if, by chance, someone knows your app's URL and you haven't created your first admin user yet, they could potentially hijack your (it's not something unique to Strapi Cloud, however it would be great to warn our users here). We have some better solution(s) for this in mind for the future.